BCStudentSoftwareDevTeam/celts
# volunteer search on the sidebar now leads to the volunteer search page

This PR fixes issue #63 

volunteer search on the sidebar now leads to the volunteer search page and highlights the Volunteer search button.

We changed the href in the sidebar.html to be `/student_search` the URL for the search page. We also added a jinja tag that sets the Volunteer Search button to be `active` if the current path is `/student_search`.

To test:
Flask run and go to any webpage with a sidebar. 
Click on Volunteer Search on the sidebar.
It should take you to the new page and highlight the button.

NOTE: The URL for the event management page doesn't exist yet so that button doesn't work. This branch is a fork from PR #62 so that should be merged before this one.
